apps/myaccount/src/components/account-recovery/account-recovery.tsx

```diff
@@ -98,6 +105,13 @@ export const AccountRecoveryComponent: FunctionComponent<AccountRecoveryProps> =
                         if (prop.name === RECOVERY_PASSWORD_NOTIFICATION) {
                             setIsNotificationRecoveryEnabled(prop.value.toLowerCase() == "true" ? true : false);
                         }
+                        if (prop.name === RECOVERY_PASSWORD_NOTIFICATION_EMAIL_LINK) {
```

Comment:
change this to use === instead. Replace all including existing implementations.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6418/commits/0dd77c8c16b11b96a91e6b1331294e3bb8560a5b